src/core/core.scale.js

```diff
@@ -832,11 +870,12 @@ var Scale = Element.extend({
 		for (i = 0; i < tickCount; i++) {
 			tick = ticks[i];
 
-			if (skipRatio > 1 && i % skipRatio > 0) {
-				// leave tick in place but make sure it's not displayed (#4635)
+			if (skipRatio <= 1 || i % skipRatio === 0) {
+				tick._index = i;
```

Comment:
I think we need to make `_index` always (even without `autoSkip`) available eventually, but this seems sufficient for now.